017-quartz-cluster/src/test/resources/common.properties

```diff
@@ -9,7 +9,7 @@ quarkus.datasource.jdbc.min-size=2
 # Quartz configuration
 quarkus.quartz.store-type=jdbc-cmt
 quarkus.quartz.clustered=true
-quarkus.quartz.start-mode=forced
```

Comment:
I didn't add a comment here with this link: https://github.com/quarkusio/quarkus/issues/15754, as start-mode is supposed to be for programmatic approach only, so this property should have not been here at first.